services/deployment-agent/docker-compose-aws.yml

```diff
@@ -0,0 +1,5 @@
+version: "3.7"
+services:
+  auto-deployment-agent:
+    dns :
+      - 8.8.8.8
```

Comment:
I don't really understand why you need this for that service? is that everywhere?